pkg/service/service.go

```diff
@@ -397,3 +399,21 @@ func isRequired(required []string, name string) bool {
 	}
 	return false
 }
+
+// MachineReadableSuccessOutput prints out machine readable output
+func MachineReadableSuccessOutput(services []occlient.Service) {
```

Comment:
Since the function is dealing with machine output, wouldn't it be helpful to keep it under the `machineoutput` package?

Reply:
No, as each output will be different.. Or else we'll run into an issue where in the machineout package, we will have multiple functions with different names :(

For example, It'd probably be named ServiceCatalogList or something along those lines, IMO doesn't make sense. But I could be wrong.

Reply:
I agree with the function name point you raised. To purely counter that point, we have adopted similar approach for auto-completion where everything lives under `pkg/odo/util/completion/completionhandlers.go`.

Besides that, I think, there's scope of avoiding duplication of code by putting it under `machineoutput` package. For example, we have renamed `Success` and `Error` structs in that package to `GenericSuccess` and `GenericError` respectively.

@kadel can you please share your thoughts?